sled-agent/src/services.rs

```diff
@@ -226,11 +215,42 @@ impl Config {
             sled_id,
             sidecar_revision,
             gateway_address,
-            all_svcs_config_path: default_services_config_path(),
+            all_svcs_ledger_path: default_services_ledger_path(),
+            storage_svcs_ledger_path: default_storage_services_ledger_path(),
         }
     }
 }
 
+// The filename of ServiceManager's internal storage.
+const SERVICES_CONFIG_FILENAME: &str = "services.toml";
+const STORAGE_SERVICES_CONFIG_FILENAME: &str = "storage-services.toml";
+
+// The default path to service configuration
+fn default_services_ledger_path() -> PathBuf {
+    Path::new(omicron_common::OMICRON_CONFIG_PATH)
+        .join(SERVICES_CONFIG_FILENAME)
+}
+
+// The default path to storage service configuration
+fn default_storage_services_ledger_path() -> PathBuf {
+    Path::new(omicron_common::OMICRON_CONFIG_PATH)
+        .join(STORAGE_SERVICES_CONFIG_FILENAME)
+}
+
+// TODO(ideas):
```

Comment:
Looks like this first part is implemented.

Reply:
I actually implemented this out fully within https://github.com/oxidecomputer/omicron/pull/2972

Reply:
Comment removed in https://github.com/oxidecomputer/omicron/pull/2946/commits/541f68dee6cdf9be61f2a0c71475ccbdccd65def